### PR TITLE
Updated bput to save time

### PR DESCRIPTION
**Pull request by dr-leustyin**

### CODE REVIEW

File: common.lic [hunk 1]
```diff
@@ -66,7 +66,7 @@ module DRC
     clear
     put message
     timer = Time.now
-    while (response = get?) || (Time.now - timer < 15)
+    while (response = get?) || (Time.now - timer < 5)
```
Comment (rpherbig) [inline common.lic]: ❌ It looks like a unit test needs to be updated too

Comment (dr-leustyin) [reply common.lic]: Test updated.  Also cleaned up the method to be a bit less spammy in practice

**Comment by Sarvatt** [comment]
wrong test, i think you want     ```assert_in_delta 5, @test.thread_variable_get('runtime'), 1.0``` in def test_bput_returns_error_code_after_timeout

**Comment by Sarvatt** [comment]
oh there were 2 failing before, that should fix the one still failing though

**Comment by Sarvatt** [comment]
changing ```assert_in_delta 15, @test.thread_variable_get('runtime'), 1.0``` to ```assert_in_delta 5, @test.thread_variable_get('runtime'), 1.0```

**Pull request merged**